modules/ossm-kiali-architecture.adoc

```diff
@@ -0,0 +1,25 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-kiali.adoc
+////
+
+[id="ossm-kiali-architecture_{context}"]
+= Kiali architecture
+
+Kiali is composed of two components: the Kiali application and the Kiali console.
+
+* *Kiali application* (back end) – This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali application does not need storage. When deploying the application to a cluster, configurations are set in ConfigMaps and secrets.
+
+* *Kiali console* (front end) – The Kiali console is a web application. 	The Kiali application serves the Kiali console, which then queries the application for data in order to present it to the user.
```

Comment:
this sentence is really hard to follow and understand. It sounds like it's a loop. "The Kiali application serves the Kiali console which then queries the application for data...."

Reply:
Revised